eclipse-pass.local.yml

```diff
@@ -39,46 +39,56 @@ services:
       - ./postgres/demo/init_postgres.sh:/docker-entrypoint-initdb.d/init_postgres.sh
 
   idp:
-    build:
-      context: ./idp
-      args:
-        TENANT: jhu
-    image: "ghcr.io/eclipse-pass/idp:${PASS_VERSION}"
+    image: "tier/shib-idp:4.3.3_20240415"
     container_name: idp
     depends_on:
-      - ldap
-    environment:
-      - JETTY_MAX_HEAP=64m
-      - JETTY_BROWSER_SSL_KEYSTORE_PASSWORD=password
-      - JETTY_BACKCHANNEL_SSL_KEYSTORE_PASSWORD=password
-    ports:
-      - 9080:${IDP_INTERNAL_PORT-8080}
+      ldap:
+        condition: service_healthy
```

Comment:
When I start up pass-docker local, it takes about 2.5 minutes to start up on my machine (mac).  With the current idp/ldap images, local startup takes about 1 minute.

It seems like the idp and ldap are ready for sometime before pass-core starts. How long does it take to start up on your machine? Waiting 2.5 minutes isn't the end of the world, but I wonder if the `depends_on` conditions could be tweaked to speed it up?

Reply:
Yes. This seems to be because the health check provided by the idp has long timeouts. It may be possible to override the health check. I will give it a try. It is probably also the case that it could depend on service_started instead since I doubt the startup depends on ldap access.

Reply:
Tested changes, and startup time is better, basically same as the startup time with the old idp/ldap config (~1 minute on mac)